### DISCUSSION TITLE
how to create cloudsearch domain using cdk version 2.51.1?

### DISCUSSION THREAD

**Discussion by kounteya29** [aws/aws-cdk#24208] [Q&A]
I didn't see a module-related cloudsearch latest CDK versions. currently, we are using the CDK 2.51.1 version of CDK. Looking for help on how to create a cloudsearch domain using CDK version 2.51.1

**Comment by cgarvis** [comment]
CloudFormation doesn't have support for CloudSearch so we can support it.  You can add a [coverage request](https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues) for CloudFormation.  

Have you looked at [AWS OpenSearch](https://aws.amazon.com/opensearch-service/)? We support it in [CDK](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_opensearchservice-readme.html)
